app/views/monitoring.html

```diff
@@ -260,6 +262,108 @@
                     </div>
                   </div>
                 </div>
+
+                <div ng-if="kindSelector.selected.kind === 'All' || kindSelector.selected.kind === 'StatefulSets'">
+                  <h2>Stateful Sets</h2>
+                  <div class="list-view-pf">
+                    <div class="list-group-item" ng-if="!(filteredStatefulSets | hashSize)">
+                      <div class="list-view-pf-main-info">
+                        <ellipsis-pulser color="dark" size="sm" msg="Loading stateful sets" ng-if="!statefulSetsLoaded"></ellipsis-pulser>
+                        <em>
+                          <div ng-if="(statefulSets | hashSize) > 0">The current filters are hiding all stateful sets.</div>
+                          <span ng-if="statefulSetsLoaded && (statefulSets | hashSize) === 0">There are no stateful sets in this project.</span>
+                        </em>
+                      </div>
+                    </div>
+                    <div class="list-group-item list-group-item-expandable"
+                         ng-repeat-start="set in filteredStatefulSets track by (set | uid)"
+                         ng-click="toggleItem($event, this, set)"
+                         ng-class="{'expanded': expanded.statefulSets[set.metadata.name]}">
+                      <div class="list-view-pf-checkbox">
+                        <button class="sr-only">{{expanded.statefulSets[set.metadata.name] ? 'Collapse' : 'Expand'}}</button>
+                        <span ng-if="expanded.statefulSets[set.metadata.name]">
+                          <span class="fa fa-angle-down"></span>
+                        </span>
+                        <span ng-if="!expanded.statefulSets[set.metadata.name]">
+                          <span class="fa fa-angle-right"></span>
+                        </span>
+                      </div>
+
+                      <div class="list-view-pf-main-info">
+                        <div class="list-view-pf-body">
+                          <div class="list-view-pf-description">
+                            <div class="list-group-item-heading">
+                              <a ng-href="{{set | navigateResourceURL}}"><span ng-bind-html="set.metadata.name | highlightKeywords : filterKeywords"></span></a>
+                              <small>created <span am-time-ago="set.metadata.creationTimestamp"></span></small>
+                            </div>
+                            <div class="list-group-item-text">
+                              <!-- TODO: is this the right status? -->
```

Comment:
probably just needs to be the replica counts like we have in the stateful sets table, we dont have a status for stateful sets